README.md

```diff
@@ -88,10 +108,11 @@ PRs welcome :)
 * New Plugin: [Optiv's Go365 Method](https://github.com/optiv/Go365) - Includes Office365 auth and userenum capabilities via SOAP
 * "Resume" functionality for paused/cancelled scans. Ideally storing data for APIs used, if they were destroyed and what user/pwd the spray was on
 * Method to reliably determine if an auth attempt was throttled, so the username could be re-queued and tried again later for full cover (would have to be per-plugin, return "throttled" boolean value in plugin script, requeue if throttled)
-* Notification system for webhooks (Teams TODO)
+* Notification system for webhooks (Always adding more)
 * Stop on success flag
 * Spray profile overhaul
 * Development notes
+* Limit MFA Sweep module to MSGraph/AZVault modules (Still Todo)
```

Comment:
Curious what you mean by this? Like only allow MFASweep if also running the MSGraph/AZVault modules? Would probably want the same ability to msol/o365? But IMO wherever creds are sprayed I think you should still be able to test them against MFASweep. Ex: If you spray an AD tied on-prem OWA server, get creds, then you want to see where that acct has access

Reply:
I meant in that if it's a MS module it'll give the option for MFASweep available rather than Okta etc :)

Reply:
IMO it should be available for anything, Okta (for example) can still be used as federated auth, I don't see why is should be exclusive ya know?